src/main/java/cinema/film/Session.java

```diff
@@ -0,0 +1,23 @@
+package cinema.film;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+public class Session {
+
+    private LocalDateTime datetime;
+    private List<Ticket> tickets;
+
+    public Session(LocalDateTime datetime, List<Ticket> tickets) {
+        this.datetime = datetime;
+        this.tickets = tickets;
+    }
+
+    @Override
+    public String toString() {

```

Comment:
please make sure that LocalDateTime is shown in correct format (7 Feb 2021, 20:00)